app/javascript/utils/Patient.js

```diff
@@ -20,19 +22,46 @@ function formatNameAlt(patient) {
   return `${patient.last_name || ''}, ${patient.first_name || ''}${patient.middle_name ? ' ' + patient.middle_name : ''}`;
 }
 
+/**
+ * Provides the yup validation for Patient phone numbers
+ * @param {Object} data - the input to validate using yup
+ */
+function phoneSchemaValidator (data) {
+  return this.test({
+    name: 'phone',
+    exclusive: true,
+    message: 'Please enter a valid Phone Number',
+    test: value => {
+      try {
+        if (!value) {
+          return true; // Blank numbers are allowed
+        }
+        // Make sure we'll be able to convert to E164 format at submission time
+        return !!phoneUtil.format(phoneUtil.parse(value, 'US'), PNF.E164) && /(0|[2-9])\d{9}/.test(value.replace('+1', '').replace(/\D/g, ''));
+      } catch (e) {
+        return false;
+      }
+    },
+  });
+}
+
 /**
  * Formats patient's phone number in E164 format.
  * @param {String} phone - patient's phone number
  */
-function formatPhoneNumber(phone) {
-  if (phone === null || phone === undefined) return '';
+function formatPhoneNumber (data) {
```

Comment:
same for `phoneSchemaValidator`

```suggestion
function formatPhoneNumber(data) {
```

Reply:
fixing now!